File: tests/extractor_fixtures/extractor_fixture.kt
// A fixture for the path extractor. See test_the_extractors_agree.py.
fun probe() {
    request("/fixture/plain", "GET")
    request("/fixture/$id/nested", "DELETE")
    val conn = (URL("$base/fixture/direct").openConnection() as HttpURLConnection).apply {
        requestMethod = "POST"
    }
}

File: tests/extractor_fixtures/extractor_fixture.swift
// A fixture for the path extractor. Its answer is written down in
// test_the_extractors_agree.py, so a capability lost here fails there rather
// than reporting a clean sweep.
func probe() async {
    _ = try? await request("/fixture/plain", method: "GET")
    _ = try? await request("/fixture/\(id)/nested", method: "DELETE")
}
